index.html

```diff
@@ -35,16 +35,16 @@
         </ul>
```

Comment:
- [ ] Please add a link to the homepage on mobile navigation

Reply:
Thanks for the professional review @Rustamxon7 . I have add a link to the to the homepage of the mobile navigation.